<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerApplyWindow</class>
 <widget class="QMainWindow" name="SellerApplyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提交认证申请</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>真实姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditRealName">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>未认证</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>证件号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIDNumber">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>未认证</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>手机号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPhone">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>未认证</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="buttonSubmitApply">
          <property name="text">
           <string>提交认证</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
